<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>401</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="newGamePushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>New game</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="numberOfDiscsLineEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="numberOfDiscsLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>55</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter the number of discs:</string>
    </property>
   </widget>
   <widget class="QLabel" name="discsErrorLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>441</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="startingTowerLineEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="startingTowerLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter the starting tower:</string>
    </property>
   </widget>
   <widget class="QLabel" name="towerErrorLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>441</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="AtowerLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>  A</string>
    </property>
   </widget>
   <widget class="QLabel" name="BtowerLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>280</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QLabel" name="CtowerLabel">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>280</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>  C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AtoBpushButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>A-&gt;B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AtoCpushButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>A-&gt;C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtoApushButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>B-&gt;A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtoCpushButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>B-&gt;C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CtoApushButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>C-&gt;A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CtoBpushButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>C-&gt;B</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNUmberMin">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>160</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberSec">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>190</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="minLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>160</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Minutes:</string>
    </property>
   </widget>
   <widget class="QLabel" name="secLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>190</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Seconds:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>140</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gametime:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeGamePushButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Close game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="solvePushButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Solve</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopSolvingPushButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop solving</string>
    </property>
   </widget>
   <widget class="QLabel" name="startingTowerAlabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>235</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="startingTowerBlabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>236</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="startingTowerClabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>240</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>733</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
